tests/rptest/tests/schema_registry_test.py

```diff
@@ -370,11 +370,18 @@ def _get_subjects_subject_versions_version(self,
 
     def _get_subjects_subject_versions_version_referenced_by(
             self, subject, version, headers=HTTP_GET_HEADERS, **kwargs):
-        return self._request(
+        deprecated = self._request(
             "GET",
             f"subjects/{subject}/versions/{version}/referencedBy",
             headers=headers,
             **kwargs)
+        standard = self._request(
+            "GET",
+            f"subjects/{subject}/versions/{version}/referencedby",
+            headers=headers,
+            **kwargs)
+        assert standard.json() == deprecated.json()
```

Comment:
non-blocker/point-of-discussion:

Do we need to check that the deprecated version works? Could we just remove it?
Based on the conversations in https://github.com/redpanda-data/redpanda/pull/12745#issuecomment-1679268405 it seems that this endpoint is not used in the field that much.

My concern here is that, in the chance where this assert fails, do we allocate engineering hours to fix the problem on a deprecated endpoint? It seems like testing noise to me.

Reply:
> non-blocker/point-of-discussion:
> 
> Do we need to check that the deprecated version works? Could we just remove it? Based on the conversations in [#12745 (comment)](https://github.com/redpanda-data/redpanda/pull/12745#issuecomment-1679268405) it seems that this endpoint is not used in the field that much.

I saw the note about older RP versions but those have reached EOL.

Reply:
I'm of the opinion that we have no way to know if it's used, the cost of keeping it low.